share/spice/forecast/forecast.js

```diff
@@ -377,9 +371,13 @@
                     // unit of measurement. If the metric setting is enabled or the user's
                     // region is not USA and the API returned temps in F, switch to 'C':
                     var uom_in_query = DDG.get_query().match(/\b(celsius|fahrenheit)\b/);
+                    var response_timezone = api_result.timezone;
                     if (uom_in_query) {
                         uom = (uom_in_query[1] === 'celsius') ? 'C' : 'F';
                         updateUnitOfMeasure();
+                    } else if(!response_timezone.match(/America/i)) {
```

Comment:
This  won't work for Canada, e.g. "America/Toronto" -- not sure we have a great way to handle this

Reply:
Fixed.